networking/p2p/src/test/java/tech/pegasys/teku/networking/p2p/DiscoveryNetworkTest.java

```diff
@@ -96,44 +93,6 @@ public void shouldStartConnectionManagerAfterP2pAndDiscoveryStarted() {
     assertThat(started).isCompleted();
   }
 
-  @Test
-  public void shouldStopConnectionManagerBeforeNetworkAndDiscovery() {
-    final SafeFuture<Void> connectionStop = new SafeFuture<>();
-    doReturn(new SafeFuture<Void>()).when(discoveryService).stop();
-    doReturn(connectionStop).when(connectionManager).stop();
-
-    discoveryNetwork.stop();
-
-    verify(connectionManager).stop();
-    verify(discoveryService).updateCustomENRField(any(), any());
-    verify(discoveryService).getEnr();
-    verifyNoMoreInteractions(discoveryService);
-    verifyNoInteractions(p2pNetwork);
-
-    connectionStop.complete(null);
-    verify(p2pNetwork).stop();
-    verify(discoveryService).stop();
-  }
-
-  @Test
-  public void shouldStopNetworkAndDiscoveryWhenConnectionManagerStopFails() {
-    final SafeFuture<Void> connectionStop = new SafeFuture<>();
-    doReturn(new SafeFuture<Void>()).when(discoveryService).stop();
-    doReturn(connectionStop).when(connectionManager).stop();
-
-    discoveryNetwork.stop();
-
-    verify(connectionManager).stop();
-    verify(discoveryService).updateCustomENRField(any(), any());
-    verify(discoveryService).getEnr();
-    verifyNoMoreInteractions(discoveryService);
-    verifyNoInteractions(p2pNetwork);
-
-    connectionStop.completeExceptionally(new RuntimeException("Nope"));
-    verify(p2pNetwork).stop();
-    verify(discoveryService).stop();
-  }
```

Comment:
We should be able to restore these tests now.

Reply:
Done.